impl/src/rendering/templates/partials/build_script.template

```diff
@@ -54,6 +54,11 @@ cargo_build_script(
         "{{flag}}",
         {%- endfor %}
     ],
+    tools = [
```

Comment:
Can you make sure this doesn't print if tools is empty?  See line 44 for an example.

Reply:
done